content/release-notes/2023-08-15-storage-and-compute.md

```diff
@@ -0,0 +1,7 @@
+### Postgres version update
+
+Supported Postgres versions were updated to 14.9 and 15.4, respectively.
+
+### BYPASSRLS added to the neon_superuser role
+
+The `neon_superuser` role now inlcudes the `BYPASSRLS` attribute, enabling members of this role to bypass the row security system when accessing tables. Roles created in the Neon console, CLI, or API, including the default role created with a Neon project, are granted membership in the `neon_superuser` role. For more information, see [The neon_superuser role](/docs/manage/roles#the-neonsuperuser-role).
```

Comment:
the projects that were created before this release, will not have such attribute.

Reply:
Fixed. Thank you.